### ISSUE TITLE
Elaborate Error: ** Fatal: C:\... : fread: The operation completed successfully.

### ISSUE THREAD

**Issue by JimLewis** [nickg/nvc#717]
I loaded commit  a56ea54.   Now the OSVVM lab regressions fail with this:  
```
nvc --std=08 -H 128m --work=Tb2_Entity:C:/tools/sim_temp/nvc-1.10-devel/TB2_ENTITY.08 -L C:/tools/sim_temp/nvc-1.10-devel -e TbMemIO
Elaborate Error: ** Fatal: C:\tools\sim_temp\nvc-1.10-devel\LIB_MEMIO.08\LIB_MEMIO.RESETSYNC-RTL: fread: The operation completed successfully.
Elaborate Error: 
Elaborate Error: 
Simulation Finish time 18:39:07, Elasped time: 0:00:00 
```

**Comment by nickg** [comment]
I think this will probably be fixed by 4147ebd7f. Could you test again?

**Comment by nickg** [comment]
Assuming this was fixed by the commit above.
